tensorboard/components/tf_data_selector/tf-data-selector.ts

```diff
@@ -40,55 +53,47 @@ Polymer({
       value: () => [],
     },
 
-    runSelectionState: {
-      type: Object,
-      observer: '_storeRunSelectionState',
-      value: () => tf_storage.getObject('runSelectionState') || {},
-    },
-
-    runRegexInput: {
+    _addLabel: {
       type: String,
-      value: tf_storage.getStringInitializer('regexInput', {defaultValue: ''}),
-      observer: '_runRegexInput',
+      computed: '_getAddLabel(_selectedExperiments)',
```

Comment:
For a case like this you could just use a computed binding: https://www.polymer-project.org/1.0/docs/devguide/data-binding#annotated-computed